resources/Materials/Examples/Syntax/GeomInfos.mtlx

```diff
@@ -5,7 +5,7 @@
   <collection name="col3" includegeom="/root/world/geo/g[89]"/>
 
   <geompropdef name="uv2" type="vector2" geomprop="texcoord" index="2"/>
-  <geompropdef name="objsize" type="float" unittype="distance" unit="meters"/>
+  <geompropdef name="objsize" type="float" unittype="distance" unit="meter"/>
```

Comment:
The Spec still has its unit examples in the plural, e.g. "meters" and "feet".  I changed them to singular ("meter" and "foot") in the draft 1.38 doc, though that feels strange to me- I prefer the plural, but I guess that ship has sailed.  Note that the two units defined for "angle" (degrees and radians) are plural: it would be nice if these were consistent.

Reply:
Good points, and let’s amend this in an upcoming update to the data libraries.